kf_release_maker/cli.py

```diff
@@ -20,19 +26,20 @@ def cli():
     pass
 
 
-@click.command(name='notes')
-@click.option('--org', default=config.DEFAULT_GH_ORG,
```

Comment:
The old version used to have the kf-release-maker as a default for the Github org. I liked that as a new user of the tool because I could see what kind of release notes it generated. I think we should keep this?

Reply:
I think that only works if your repo has unreleased PRs though